phoenix-core/src/main/java/org/apache/phoenix/query/QueryConstants.java

```diff
@@ -40,6 +37,125 @@
 import org.apache.phoenix.schema.SystemStatsSplitPolicy;
 import org.apache.phoenix.schema.TableProperty;
 
+import static org.apache.phoenix.jdbc.PhoenixDatabaseMetaData.APPEND_ONLY_SCHEMA;
```

Comment:
Most of these changes are also after passing through a linter which pointed out that `public static final` is unnecessary since this is an interface.